src/templates/partials/languages/sh.html

```diff
@@ -23,22 +23,23 @@
 <!-- Translations: Serbo-Croatian -->
 {% macro t(key) %}{{ {
   "language": "sh",
+  "direction": "ltr",
```

Comment:
This is not needed – it's the default and given by the `en` translation, please remove.